### PR TITLE
UI: fix login on UI

### PR DESCRIPTION
**Pull request by Pearl1594**
### Description
Fixes https://github.com/apache/cloudstack/issues/4908

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [X] Major
- [ ] Minor
- [ ] Trivial

### How Has This Been Tested?
Specify username in the following format : xxx@domain.com and verified the login parameters being passed. Username is no longer undefined but points to the passed email address.

Without fix:
image

With Fix:
image

### CODE REVIEW

**Comment by Pearl1594** [comment]
@blueorangutan ui

**Review by shwstppr — APPROVED** [review]

**Comment by alexandremattioli** [comment]
@blueorangutan package

**Review by rhtyd — APPROVED** [review]

File: ui/src/api/index.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ export function api (command, args = {}, method = 'GET', data = {}) {
 export function login (arg) {
   const params = new URLSearchParams()
   params.append('command', 'login')
-  params.append('username', arg.username)
+  params.append('username', arg.username || arg.email)
```
Comment (rhtyd) [inline ui/src/api/index.js:45]: The culprit is in Login.vue, but this is okay.

Comment (rhtyd) [reply ui/src/api/index.js:45]: https://github.com/apache/cloudstack/blob/master/ui/src/views/auth/Login.vue#L182

**Comment by rhtyd** [comment]
Tested LGTM

**Pull request merged**